x-pack/plugins/cases/public/components/markdown_editor/editor.tsx

```diff
@@ -26,8 +33,18 @@ interface MarkdownEditorProps {
 export const { uiPlugins, parsingPlugins, processingPlugins } = {
   uiPlugins: getDefaultEuiMarkdownUiPlugins(),
   parsingPlugins: getDefaultEuiMarkdownParsingPlugins(),
-  processingPlugins: getDefaultEuiMarkdownProcessingPlugins(),
+  processingPlugins: getDefaultEuiMarkdownProcessingPlugins() as [
```

Comment:
This is a doozy

Reply:
yep, should be removed eventually: https://github.com/elastic/eui/issues/4688